tests/container_disk_test.go

```diff
@@ -83,9 +84,8 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 	Describe("[rfe_id:273][crit:medium][vendor:cnv-qe@redhat.com][level:component]Starting and stopping the same VirtualMachineInstance", func() {
 		Context("with ephemeral registry disk", func() {
 			It("[test_id:1463][Conformance] should success multiple times", func() {
```

Comment:
Please consider dropping this test in a follow-up. We have oh-so-many tests running with ephemeral disks, I don't see the value of this specific one.